include/mega/types.h

```diff
@@ -262,6 +263,27 @@ typedef enum { NOTLOGGEDIN, EPHEMERALACCOUNT, CONFIRMEDACCOUNT, FULLACCOUNT } se
 // in a 64-bit int
 typedef uint64_t handle;
 
+class NodeHandle
+{
+    // Handles of nodes are only 6 bytes.
+    // This class helps avoid issues when we don't save/restore the top 2 bytes when using an 8 byte uint64 to represent it
+    uint64_t h = 0xFFFFFFFFFFFFFFFF;
```

Comment:
Are we planning to get rid of the `UNDEF` at somepoint? otherwise, I think that we should use it here:
```suggestion
    uint64_t h = UNDEF;
```
However, it might be more clear to indicate its `0xFFFFFFFFFFFFFFFF` :thinking:

Reply:
I don't think this matters - either way is fine.   The FFs might make comparison with the other constants a little bit easier

Reply:
And, we will keep UNDEF for the 8 byte handles